CHANGELOG.md

```diff
@@ -1,3 +1,74 @@
+CHANGES IN VERSION 1.19.0 (FROM 1.18.3)
+===================================
+
+Appium 1.19.0 is a minor release
+
+### Android General
+
+* Add capabilities:
+  * `allowDelayAdb` to enable or disable `'-delay-adb` option on emulator startup if the options was supported
```

Comment:
The capability prevents Appium from using `-delay-adb` command line option to detect emulator startup in certain scenarios. See https://github.com/appium/appium/issues/14773 for more details